webviz_subsurface/containers/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -265,23 +270,32 @@ def _render_lines(ensemble, vector):
                 for r, df in data.groupby(["REAL"])
             ]
             traces[0]["hoverinfo"] = "x"
+
+            # Check if a data has been clicked previously
+            # If so, add the vertical line to the figure
+            if date:
+                ymin = min([min(trace["y"]) for trace in figure["data"]])
+                ymax = max([max(trace["y"]) for trace in figure["data"]])
+                date = json.loads(date)
+                layout = {
+                    "shapes": [
+                        {"type": "line", "x0": date, "x1": date, "y0": ymin, "y1": ymax}
+                    ],
+                    "showlegend": False,
+                }
+            else:
+                layout = {
+                    "title": "Click on a date to calculate tornado plot. "
+                    + "Click on a bar in tornadoplot to highlight relevant realizations",
```

Comment:
Could this be removed now, and instead rely on the tour to explain interaction behavior?